.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+{
+  name: "Release",
+  on: {
+    release: {
+      types: [ "created" ]
+    }
+  },
+  jobs: {
+    publish: {
+      runs-on: "ubuntu-latest",
+      steps: [
+        {
+          uses: "actions/checkout@v2",
+          with: {
+            ref: "${{ env.GITHUB_REF }}"
+          }
+        },
+        {
+          name: "Set up node 12",
+          uses: "actions/setup-node@v2",
+          with: {
+            registry-url: "https://registry.npmjs.com",
+            node-version: "12"
+          }
+        },
+        { run: "yarn" },
+        { run: "yarn lerna run compile" },
+        {
+          run: "node_modules/.bin/lerna publish from-package",
```

Comment:
I had to publish this way directly from the commandline, instead of `yarn lerna …`.  It seems `yarn` got in the way 🤷